GiniVision/Classes/GiniConfiguration.swift

```diff
@@ -109,6 +109,13 @@ import UIKit
     
     
     // MARK: Camera options
+    
+    /**
+     Toggles file import feature.
```

Comment:
Toggles *the* import feature.
Also, only 4 words might not be enough for people to understand the parameter completely. And since the configuration is user facing, it might make sense to explain it a little better.